docs/sgp40_voc_index/arduino.rst

```diff
@@ -107,6 +107,13 @@ VOC Index is zero
 During the initialization phase of the VOC Index algorithm, which is the first
 100 iterations, the VOC Index will output zero.
 
+Arduino Nano 33 BLE doesn't show up in port list
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
The title underline must be equal the length of the title for strict restructured text compatibility:
```
System Message: WARNING/2 (/home/rnestler/projects/github/embedded-sgp/docs/sgp40_voc_index/arduino.rst, line 111)
Title underline too short.
Arduino Nano 33 BLE doesn't show up in port list
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Maybe we should add a rst linter to the CI.

Reply:
...also the column wrapping seems >80chars